packages/common/src/api/tan-query/jupiter/utils.ts

```diff
@@ -33,6 +37,8 @@ import {
   UserBankManagedTokenInfo
 } from './types'
 
+const USDC_MINT_ADDRESS = 'EPjFWdd5AufqSSqeM2qN1xzybapC8G4wEGGkZwyTDt1v'
```

Comment:
nit: maybe pull this from env? I know it doesn't really matter but just so we don't need multiple constants lying around